scripts/init-env.sh

```diff
@@ -57,10 +57,15 @@ create_multus_annotation(){
     MULTUS_ANNOTATION="${MULTUS_ANNOTATION}{ \"name\" : \"${NET_NAME}-$1-${NUM}\" },"
   done
 }
-# IPv4
-create_multus_annotation "ipv4"
-# IPv6
-create_multus_annotation "ipv6"
+
+# Only add annotation in non OCP clusters
+if $TNF_NON_OCP_CLUSTER
+then
+  # IPv4
+  create_multus_annotation "ipv4"
+  # IPv6
+  create_multus_annotation "ipv6"
+fi
```

Comment:
Potentially we could add some logging as to what annotations are being created.

```suggestion
if [ -z $TNF_NON_OCP_CLUSTER ]; then
   echo 'TNF_NON_OCP_CLUSTER is unset'
else
   echo 'creating multus annotations'

  # IPv4
  create_multus_annotation "ipv4"
  # IPv6
  create_multus_annotation "ipv6"
fi
```

Reply:
Fixed: Added a log for creating annotation. TNF_NON_OCP_CLUSTER is a boolean so kept the previous comparison